test/utils.js

```diff
@@ -111,7 +111,10 @@ function setOptions() {
   // visit URL(http://domain-name/test/index.html?prefer=fast/sustained/low)
   var parameterStr = window.location.search.substr(1);
   var reg = new RegExp("(^|&)prefer=([^&]*)(&|$)", "i");
+  // use webgpu=true parameter for WebGPU backend
+  var webgpu_reg = new RegExp("(^|&)webgpu=([^&]*)(&|$)", "i");
   var r = parameterStr.match(reg);
+  var webgpu_r = parameterStr.match(webgpu_reg);
```

Comment:
FYI 

Regular regressions and string splitting to pull out query parameters from the URL is a traditional way. I prefer to use `URLSearchParams` for retrieving query string variables. Usually, the regex method will be much slower although it's sensitive in this case.

https://github.com/intel/webml-polyfill/blob/master/examples/util/base.js has the same issue but I don't expect we will change it.

const parseSearchParams = (key) => {
  let searchParams = new URLSearchParams(location.search);
  return searchParams.get(key);
};

const hasSearchParam = (key) => {
  let searchParams = new URLSearchParams(location.search);
  return searchParams.has(key);
};

Reply:
@ibelem I have updated pr, please take a review again, thanks.